unit_tests/log_repl/test_main_log_repl_atomic_helper.cpp

```diff
@@ -1114,3 +1114,15 @@ log_rv_check_redo_is_needed (const PAGE_PTR & /*pgptr*/, const LOG_LSA & /*rcv_l
   assert_release (false);
   return false;
 }
+
+bool
+is_passive_transaction_server()
+{
+  return false;
```

Comment:
I think
```suggestion
  assert (false);
  return false;
```

same below ...

Reply:
OK,  I will add assertion

Reply:
Also make sure that the unit tests pass.